mesonbuild/mdist.py

```diff
@@ -327,9 +327,6 @@ def run(options: argparse.Namespace) -> int:
     b = build.load(options.wd)
     need_vsenv = T.cast('bool', b.environment.coredata.get_option(OptionKey('vsenv')))
     setup_vsenv(need_vsenv)
-    # This import must be load delayed, otherwise it will get the default
-    # value of None.
-    from mesonbuild.mesonlib import get_meson_command
     src_root = b.environment.source_dir
```

Comment:
Can you mention commit 23d3b98fc1ed3b774cc3838da89b2e8f0f91800b in the commit message for this?